corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -67,10 +82,26 @@ def add_arguments(self, parser):
             help="If existing progress files for this command exist, turn this flag on to ignore them and start"
                  "from scratch.",
         )
+        parser.add_argument(
+            '--only-run-once',
```

Comment:
Maybe call this `--force` so that the default is to only run once and if you want to run it again you have to pass in `--force`?